### PR TITLE
prev/next updates and other adjustments

### PR DESCRIPTION
**Pull request by zekiel**

### CODE REVIEW

**Comment by zekiel** [comment]
//cc @moollaza @talsraviv

File: duckduckhack/ddh-prev-next.json [hunk 1]
```diff
@@ -141,37 +141,32 @@
...
 		"prev" : ["testing_location_language_apis.md"],
-		"next" : ["review_process.md"]
+		"next" : ["metadata.md"]
 	},
 
-	"review_process.md": {
-    	"prev": ["preparing_for_a_pull_request.md"],
-    	"next": ["developer_checklist.md"]
+	"metadata.md": {
+		"prev": ["preparing_for_a_pull_request.mdd"],
```
Comment (moollaza) [inline duckduckhack/ddh-prev-next.json]: small typo here `.mdd`

File: duckduckhack/submitting-your-instant-answer/submission_and_review.md [hunk 2]
```diff
@@ -23,3 +34,7 @@ Here's how the whole process looks, after you submit your pull request:
 2. When the community leader feels that the Instant Answer is ready to go live, they ping [@jagtalon](https://github.com/jagtalon) so that he can set it up on a testing server.
 3. Final review by the internal team.
 4. Your pull request gets merged in and goes live!
+
+## Thank you!  
+
+We're excited to have you in our community and we can't wait to see your Instant Answer on DuckDuckGo.com. Your contributions are helping our search engine get better every day. We owe you our deepest gratitude.
```
Comment (moollaza) [inline duckduckhack/submitting-your-instant-answer/submission_and_review.md]: I typically tell them to "stick around" too?

>Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Comment by moollaza** [comment]
LGTM :+1: 

Made one small suggestion^

**Pull request merged**